source/faq.txt

```diff
@@ -37,3 +37,19 @@ In MongoDB deployments with mixed versions of :binary:`~bin.mongod`, it is
 possible to get an ``Unrecognized pipeline stage name: '$sample'``
 error. To mitigate this situation, explicitly configure the partitioner
 to use and define the Schema when using DataFrames.
+
+How can I use mTLS for authentication?
+--------------------------------------
+
+Include the following options when you run ``spark-submit``:
+
+.. code-block:: bash
+
+   --driver-java-options -Djavax.net.ssl.trustStore=<path to your truststore.jks file> \
+   --driver-java-options -Djavax.net.ssl.trustStorePassword=<your truststore password> \
+   --driver-java-options -Djavax.net.ssl.keyStore=<path to your keystore.jks file> \
+   --driver-java-options -Djavax.net.ssl.keyStorePassword=<your keystore password> \
+   -conf spark.executor.extraJavaOptions=-Djavax.net.ssl.trustStore=<path to your truststore.jks file> \
+   -conf spark.executor.extraJavaOptions=-Djavax.net.ssl.trustStorePassword=<your truststore password> \
+   -conf spark.executor.extraJavaOptions=-Djavax.net.ssl.keyStore=<path to your keystore.jks file> \
+   -conf spark.executor.extraJavaOptions=-Djavax.net.ssl.keyStorePassword=<your keystore password> \
```

Comment:
Q: What are the backslashes for at the end of each line? Could you confirm with an engineer if these are needed or not?

Reply:
When you call `spark-submit` from a terminal, you need to pass these options on a single line. To avoid insanely long lines, you can use a backslash to show line continuation. So if a user wanted to use these options from a real command line, they could copy this sample in and it should (theoretically) work.